routing/routing_consistency_tests/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ omim_link_libraries(
   osrm
   jansson
   protobuf
+  mwm_diff
```

Comment:
```shell
y@lambda:~/coding/omim/generator/mwm_diff$ git grep -h '#include' | sort | uniq
#include "3party/bsdiff-courgette/bsdiff/bsdiff.h"
#include "base/logging.hpp"
#include "base/scope_guard.hpp"
#include <boost/python.hpp>
#include "coding/file_name_utils.hpp"
#include "coding/file_reader.hpp"
#include "coding/file_writer.hpp"
#include "coding/internal/file_data.hpp"
#include "coding/reader.hpp"
#include "coding/writer.hpp"
#include "coding/write_to_sink.hpp"
#include "coding/zlib.hpp"
#include <cstdint>
#include "generator/mwm_diff/diff.hpp"
#include <iterator>
#include "platform/platform.hpp"
#include <string>
#include "testing/testing.hpp"
#include <vector>
```
Therefore, mwm_diff must be listed before coding, base and generator. Otherwise correct linking is not guaranteed.

Reply:
исправила, поставила непосредственно после storage, который является единственным местом использования mwm_diff, до base, coding, generator